frontend/app/scripts/controllers/login.js

```diff
@@ -12,7 +12,7 @@ angular.module('frontendApp')
       AuthenticationService.Login($scope.credentials, function (response) {
         if (response) {
           AuthenticationService.SetCredentials(response.name, response.details.sessionId, true);
-          if($rootScope.locationPath !== "/login" && $rootScope.locationPath !==""){
+          if($rootScope.locationPath !== "/login" && $rootScope.locationPath !=="" && $rootScope.locationPath){
```

Comment:
````javascript
if ( $rootScope.locationPath && $rootScope.locationPath !== "/login") {
````
If you check $rootScope.locationPath, then next check $rootScope.locationPath !== "" not needed